docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,201 @@
+---
+id: reanimated-drawer-layout
+title: Reanimated Drawer Layout
+sidebar_label: Reanimated Drawer Layout
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cross-platform replacement for the React Native's [DrawerLayoutAndroid](http://reactnative.dev/docs/drawerlayoutandroid.html) component.  
+For detailed usage of standard parameters, please refer to the [React Native docs](http://reactnative.dev/docs/drawerlayoutandroid.html).
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import ReanimatedDrawerLayout from 'react-native-gesture-handler/ReanimatedDrawerLayout';
+```
+
+## Properties:
+
+### `drawerType`
+
+specifies the way the drawer will be displayed.
+Accepts values of the `DrawerPosition` enum. Defaults to `FRONT`.
+
+- `FRONT` the drawer will be displayed above the content view.
+- `BACK` the drawer will be displayed below the content view, revealed by sliding away the content view.
+- `SLIDE` the drawer will appear attached to the content view, opening it slides both the drawer and the content view.
+
+| `FRONT`                                               | `BACK`                                               | `SLIDE`                                               |
+| ----------------------------------------------------- | ---------------------------------------------------- | ----------------------------------------------------- |
+| <img src={useBaseUrl('gifs/new-drawer-front.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-back.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-slide.gif')} /> |
+
+### `edgeWidth`
+
+width of the invisible, draggable area on the edge of the content view, which can be dragged to open the drawer.
+
+### `hideStatusBar`
+
+a boolean value. When set to `true`, drawer component will use [StatusBar API](http://reactnative.dev/docs/statusbar.html) to hide the OS status bar when the drawer is dragged or idle in the `open` position.
+
+### `statusBarAnimation`
+
+a string with possible values: `slide`, `none` or `fade`. Defaults to `slide`.
+May be used in combination with `hideStatusBar` to select the animation used for hiding the status bar.
+See [StatusBar API](http://reactnative.dev/docs/statusbar.html#statusbaranimation) docs.
+
+### `overlayColor`
+
+color of the background overlay on top of the content window when the drawer is `open`.  
+This color's opacity animates from 0% to 100% as the drawer transitions from closed to open. Defaults to `rgba(0, 0, 0, 0.7)`.
+
+### `renderNavigationView`
+
+a renderer function for the drawer component, provided with a `progress` parameter.
+
+-  `progress` - `SharedValue` that indicates the progress of drawer opening/closing animation.
+    - equals `0` when the `drawer` is closed and `1` when the `drawer` is opened
```

Comment:
Cambridge dictionary says it should be `open` when the word is used as an adjective, and `opened` when it's used as a past/present/future perfect verb.
In this context, it could very well be either a verb, or an adjective, and so both forms are likely valid.